src/protocol.rs

```diff
@@ -246,6 +231,11 @@ impl SignatureShare {
         }
     }
 
+    /// Create a new [`SignatureShare`] given (r, s).
```

Comment:
suggestion: what do `r` and `s` actually represent? It actually might be complicated to explain that here so I'd suggest just ending the sentence after ```[`SignatureShare`].```